packages/account/src/Components/file-uploader-container/file-uploader-container.jsx

```diff
@@ -1,10 +1,45 @@
 import React from 'react';
-import { Icon } from '@deriv/components';
-import { Localize } from '@deriv/translations';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
+import { PlatformContext, isDesktop } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
 import { WS } from 'Services/ws-methods';
 import FileUploader from './file-uploader.jsx';
 
+const FileProperties = () => {
+    const properties = [
+        { name: 'size', icon: 'IcPoaFileEightMb', text: 'Less than 8MB' },
+        { name: 'format', icon: 'IcPoaFileFormat', text: 'JPEG  JPG  PNG  PDF  GIF' },
+        { name: 'time', icon: 'IcPoaFileTime', text: '1 - 6 months old' },
+        { name: 'clear', icon: 'IcPoaFileClear', text: 'A clear colour photo or scanned image' },
+        {
+            name: 'with-address',
+            icon: 'IcPoaFileWithAddress',
+            text: 'Issued under your name with your current address',
+        },
+    ];
+    return (
+        <div className='account-poa__upload-property'>
+            {properties.map(item => (
+                <div
+                    className={`account-poa__upload-property-item account-poa__upload-property-${item.name}`}
+                    key={item.name}
+                >
+                    <div className='account-poa__upload-property-wrapper'>
+                        <Icon icon={item.icon} className='account-poa__upload-icon-dashboard' size={40} />
+                        <Text size='xxxs' weight='bold' align='center'>
+                            {localize(item.text)}
```

Comment:
`localize` doesn't work this way. We should have the string explicitly passed as an argument.
Passing the string with a variable won't work.